src/Engine/ProtoCore/Lang/BuiltInFunctionEndPoint.cs

```diff
@@ -2581,24 +2581,9 @@ public StackValueComparerForDouble(bool ascending)
             mbAscending = ascending;
         }
 
-        private static bool Equals(StackValue sv1, StackValue sv2)
-        {
-            bool sv1null = !sv1.IsNumeric;
-            bool sv2null = !sv2.IsNumeric; 
-            if ( sv1null && sv2null)
-                return true;
-            if (sv1null || sv2null)
-                return false;
-
-            var v1 = sv1.IsDouble ? sv1.DoubleValue: sv1.IntegerValue;
-            var v2 = sv2.IsDouble ? sv2.DoubleValue: sv2.IntegerValue;
-
-            return MathUtils.Equals(v1, v2);
-        }
-
         public int Compare(StackValue sv1, StackValue sv2)
         {
-            if (Equals(sv1, sv2))
+            if (!sv1.IsNumeric && !sv2.IsNumeric)
```

Comment:
what does this do? It looks like any items that are not numeric are evaluated as equal?

Reply:
This check was being made in the previous `Equals` method that has been removed.